includes/definitions/webmon.yaml

```diff
@@ -0,0 +1,10 @@
+os: webmon
+text: 'Webmon Edge'
+type: environment
+icon: dantel
```

Comment:
Hi,
You should include here a "mib: webmon" statement, and move your mib file to a webmon folder. doing this would allow better sorting and avoid unnecessary load of the webmon mib for other devices. 
The logic of mib loading is : 
- Load all files in mibs folder (as they are supposed to be "standard" mib files)
- Load all OS specific files from mibs/<folder>/ and <folder> is defined with the "mibs:" statement of the YAML. 

Bye